src/router/index.ts

```diff
@@ -87,7 +87,7 @@ export const routesViews: Array<RouteConfig> = [
         meta: { authorize: [], sidebar: true }
       },
       {
-        path: 'coordinates/:xCoord/:yCoord',
+        path: 'coordinates/:longitude/:latitude',
```

Comment:
A short internet search puts latitude before longitude

Reply:
The most important is what the user of this would expect / want to see.
Currently this follows the order used by the mapbox internals but we could always work around this.

Reply:
We can keep it like this for now, awaiting the clients feedback